view/base/index.ts

```diff
@@ -1,6 +1,15 @@
+import { isHotkey } from '../../lib/hotkeys.js'
 import { accent } from '../../stores/accent.js'
+import { redo, undo } from '../../stores/history.js'
 import { loading } from '../../stores/loading.js'
 
+const IS_MAC = /Mac|iPod|iPhone|iPad/.test(window.navigator.platform)
+
+const HOTKEYS = {
+  redo: IS_MAC ? ['meta+shift+z'] : ['ctrl+shift+z', 'ctrl+y'],
```

Comment:
Objects are bad for JS bundles because minifier can't minify keys.

Here it is better to use variable.

Reply:
oh, ok. done